rover/usbreset.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
why a python and a c version are both needed? How is the C version compiled. 

Could you put them in a package folder please